packages/core/src/Stores/notification-store.js

```diff
@@ -161,11 +161,7 @@ export default class NotificationStore extends BaseStore {
         if (LocalStore.get('active_loginid') !== 'null')
             this.resetVirtualBalanceNotification(LocalStore.get('active_loginid'));
 
-        if (window.location.pathname === routes.cashier_p2p) {
-            this.notification_messages = this.notification_messages.filter(
-                notification => notification.platform === 'P2P'
-            );
```

Comment:
This also filters out other notifications such as `2_f_a` and `poa_poi_verified` which is previously shown in P2P, thus the notification count would decrement by `2` when a new completed order notification is shown